components/core/Button.tsx

```diff
@@ -1,22 +1,66 @@
 import { chakra, Button as ChakraButton } from "@chakra-ui/react";
+import theme from "@/src/theme";
+
 
 export const Button = chakra(ChakraButton, {
```

Comment:
@sttic can we delete this button since this was originally just an example?

Reply:
I think it's ok to keep in! It's a general button with just the most basic styles that others can take from. It's kinda like https://github.com/uOttaHack/2021-landing-page/blob/develop/components/core/Button.tsx where there's a base button that specifies stuff like font and border radius that others can take from so it doesn't need to be repeated everywhere